end_to_end/0-AutoClaimFraudDetection.ipynb

```diff
@@ -0,0 +1,792 @@
+{
```

Comment:
<p>Overall, this 0th notebook should be the overview of 5 notebooks.</p><p>However, in this notebook you started jumping into data exploration.</p><p>Suggesting to either rename the title, or restructure the notebook content with a table of content at the top.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Great Idea. I will rename to Overview and Exploratory Data Visualization